pyproject.toml

```diff
@@ -22,6 +22,9 @@ build-schema.help = "Build GraphQL schema to saleor/graphql/schema.graphql file"
 migrate.cmd = "python manage.py migrate"
 migrate.help = "Run database migrations"
 
+make-migrations.cmd = "python manage.py makemigrations"
+make-migrations.help = "Create new migrations based on changes in models"
+
```

Comment:
I don't see the benefit of calling `poe make-migrations` instead of `./manage.py makemigrations`

Reply:
Why? I like that I have common commands in poe instead of having some in manage.py and some in poe 🤔